### DISCUSSION TITLE
群晖docker出现意外停止和内存溢出问题

### DISCUSSION THREAD

**Discussion by chenbin3625** [messense/aliyundrive-webdav#423] [Q&A] [answered]
### 问题描述

<img width="1102" alt="image" src="https://user-images.githubusercontent.com/56670467/167232019-f216592d-c39c-45d4-bfc4-41968db13c35.png">

一共是4G内存，启动容器前占用约800M。
已经限制容器内存1G，但是还是会因为内存不足而停止运行。

### 重现步骤

1.启动容器
2.开启cloud sync同步数据

### 版本

latest

### 运行平台

群晖docker x86

### 日志

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/56670467/167232058-d115b042-25bc-46ec-a052-4d4e577e55df.png">

**Comment by messense** [comment, marked as answer]
没有群晖，测试不了。

请提供更详细的信息比如并发上传数等，另外日志请提供完整文本而不是截图。

**Reply by chenbin3625** [reply]
感谢回复。并发同步数之前设置为了20，现在修改为最低的3，已经正常了。
